integration-cli/docker_cli_run_unix_test.go

```diff
@@ -196,6 +196,75 @@ func (s *DockerSuite) TestRunAttachDetachFromFlag(c *check.C) {
 	c.Assert(running, checker.Equals, "true", check.Commentf("expected container to still be running"))
 }
 
+// TestRunDetach checks attaching and detaching with the escape sequence specified via flags.
+func (s *DockerSuite) TestRunAttachDetachFromInvalidFlag(c *check.C) {
```

Comment:
Is it just a new integration test for checking the warning ? (if yes, why not using a existing one ?)